src/components/chart/chart.tsx

```diff
@@ -0,0 +1,377 @@
+import { Component, h, Prop, Watch } from '@stencil/core';
+import { Languages } from '../date-picker/date.types';
+import translate from '../../global/translations';
+import { createRandomString } from '../../util/random-string';
+import { ChartItem } from './chart.types';
+
+const PERCENT = 100;
+const DEFAULT_AXIS_INCREMENT = 10;
+
+/**
+ * A chart is a graphical representation of data, in which
+ * visual symbols such as such bars, dots, lines, or slices, represent
+ * each data point, in comparison to others.
+ *
+ * @exampleComponent limel-example-chart-stacked-bar
+ * @exampleComponent limel-example-chart-orientation
+ * @exampleComponent limel-example-chart-max-value
+ * @exampleComponent limel-example-chart-type-bar
+ * @exampleComponent limel-example-chart-type-scatter
+ * @exampleComponent limel-example-chart-type-area
+ * @exampleComponent limel-example-chart-type-line
+ * @exampleComponent limel-example-chart-type-pie
+ * @exampleComponent limel-example-chart-type-doughnut
+ * @exampleComponent limel-example-chart-type-ring
+ * @exampleComponent limel-example-chart-type-gantt
+ * @exampleComponent limel-example-chart-multi-axis
+ * @exampleComponent limel-example-chart-multi-axis-with-negative-start-values
+ * @exampleComponent limel-example-chart-multi-axis-area-with-negative-start-values
+ * @exampleComponent limel-example-chart-axis-increment
+ * @exampleComponent limel-example-chart-accessibility
+ * @exampleComponent limel-example-chart-styling
+ * @exampleComponent limel-example-chart-creative-styling
+ * @beta
+ */
+
+@Component({
+    tag: 'limel-chart',
+    shadow: true,
+    styleUrl: 'chart.scss',
+})
+export class Chart {
+    /**
+     * Defines the language for translations.
+     * Will translate the translatable strings on the components.
+     */
+    @Prop({ reflect: true })
+    public language: Languages = 'en';
+
+    /**
+     * Helps users of assistive technologies to understand
+     * the context of the chart, and what is being displayed.
+     */
+    @Prop({ reflect: true })
+    public accessibleLabel?: string;
+
+    /**
+     * Helps users of assistive technologies to understand
+     * what the items in the chart represent.
+     */
+    @Prop({ reflect: true })
+    public accessibleItemsLabel?: string;
+
+    /**
+     * List of items in the chart,
+     * each representing a data point.
+     */
+    @Prop()
+    public items!: ChartItem[];
+
+    /**
+     * Defines how items are visualized in the chart.
+     */
+    @Prop({ reflect: true })
+    public type?:
+        | 'area'
+        | 'bar'
+        | 'doughnut'
+        | 'line'
+        | 'pie'
+        | 'ring'
+        | 'scatter'
```

Comment:
If we can get a real example for the `scatter` type to work we can keep this, otherwise we should remove it for now

Reply:
I'll rename this current `scatter` to `dot`, and will add scatter separately

Reply:
I think this is the correct term, or at least the best that I find.

shot from the same page: 
<img width="1024" alt="image" src="https://github.com/user-attachments/assets/52fcfcf8-360e-4678-b694-05480d9330fa">

 We could also call them `point chart` but that' just a "meh…😑 "